core/templates/dev/head/library/library.html

```diff
@@ -171,7 +187,7 @@ <h2 class="oppia-library-group-header">
 
 {% block footer %}
   <div ng-controller="LibraryFooter">
-    <div ng-if="!inSearchMode">
+    <div ng-if="!inSearchMode && !inRankMode">
```

Comment:
Is rank-mode infinite scroll? If not, you can leave the footer around in that case.